web-service/README.md

```diff
@@ -1,40 +1,94 @@
 # Physical Web Service
 
-This is an App Engine project which implements a sample Physical Web "metadata" service.  The use of a Physical Web Service is not necessary, but remarkably useful.  It helps clients resolve URLs in a safer and more efficient manner (read below for more details).
+This is an App Engine project which implements a sample Physical Web "metadata"
+service.  The use of a Physical Web Service is not necessary, but remarkably
+useful.  It helps clients resolve URLs in a safer and more efficient manner
+(read below for more details).
+
+This particular implementation is made to work with the sample Physical Web
+applications in this repo.
 
-This particular implementation is made to work with the sample Physical Web applications in this repo.
 
 ## Why do we use a Physical Web Service?
 
-UriBeacon devices have an open but very specific protocol which must be followed.  Anyone is free to do so.  This is the only requirement for participation and interaction with the Physical Web.
+UriBeacon devices have an open but very specific protocol which must be
+followed.  Anyone is free to do so.  This is the only requirement for
+participation and interaction with the Physical Web.
+
+Any app/device can read these UriBeacon packets and do with them whatever they
+wish.  We have provided some example apps (and published them to app stores) to
+make it easier for you to try out the Physical Web.
 
-Any app/device can read these UriBeacon packets and do with them whatever they wish.  We have provided some example apps (and published them to app stores) to make it easier for you to try out the Physical Web.
+Our app of course wants to provide a beautiful UX, with rich information about
+the URLs it finds, but even more so, it wants to protect the user and his/her
+privacy from the many potential misuses of the physical web.  We hope every
+other app will do so, also!
 
-Our app of course wants to provide a beautiful UX, with rich information about the URLs it finds, but even more so, it wants to protect the user and his/her privacy from the many potential misuses of the physical web.  We hope every other app will do so, also!
+One really nice property of the Physical Web is that UriBeacons cannot
+physically detect when clients scan it (and so cannot track passers-by).
+However, the web servers actually backing these URLs can of course track all
```

Comment:
Simplify sentence to: "However, the web servers backing these URL's can track all requests to them (as per..."